net/haproxy/src/opnsense/mvc/app/controllers/OPNsense/HAProxy/forms/dialogUser.xml

```diff
@@ -0,0 +1,26 @@
+<form>
+    <field>
+        <id>user.enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help>Enable this user.</help>
+    </field>
+    <field>
+        <id>user.name</id>
+        <label>Name</label>
+        <type>text</type>
+        <help>Name to identify this user.</help>
+    </field>
+    <field>
+        <id>user.description</id>
+        <label>Description</label>
+        <type>text</type>
+        <help>Description for this user.</help>
+    </field>
+    <field>
+        <id>user.password</id>
+        <label>Password</label>
+        <type>password</type>
+        <help><![CDATA[Both secure (encrypted) and insecure (unencrypted) passwords can be used. Most systems support MD5, SHA-256, SHA-512, and, of course, the classic DES-based method of encrypting passwords.<br/><div class="text-info"><b>NOTE:</b> Avoid using unencrypted passwords that start with a $-sign, because this indicates an encrypted password and will make it impossible to authenticate.</div>]]></help>
```

Comment:
They are not secure for passwords because password hashes should be hard to calculate. This are file integrity hashes. MD5 and SHA1 are not even considered to be secure because they are not collision safe.

Secure Password hashes are currently for example PBKDF2, Argon 2, bcrypt and scrypt.